### ISSUE TITLE
Wrong cursor position

### ISSUE THREAD

**Issue by MatanRubin** [mwxgaf/obsidian-dynamic-rtl#20]
Hi, when writing RTL text I noticed that the cursor position is not where I expect it to be - it hangs at the beginning of the line on the right side, instead of at the end of the line on the left side. 

Is there any way to resolve this?

Thanks in advance for your help and for your work on this!

**Comment by mwxgaf** [comment]
Hi,
Unfortunately this is the biggest unsolved problem for this plugin so far.
Here is the solution with a trick :point_right: [See this post in Obsidian forum](https://forum.obsidian.md/t/mixed-rtl-and-ltr/34808/31?u=mwxgaf)
I explained my problem to solve this kind of issues in [this comment](https://github.com/mwxgaf/obsidian-dynamic-rtl/issues/18#issuecomment-1326763869)

**Comment by MatanRubin** [comment]
Yes, I understand the difficulty involved. Is this something you think the obsidian devs could easily help with? If there’s a simple ask that would help with this, we can reach out to them and I’m sure they’d be happy to help. I would’ve asked them myself, but I’m not sure what I should ask for…

**Comment by mwxgaf** [comment]
I almost explained the main question in the link that I referenced in the previous comment, first link actually.
No I haven't asked yet.
